AdobeStockImageAdminUi/Test/Mftf/Test/UserSearchesStockImagesByKeywords.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="UserSearchesStockImagesByKeywords">
+        <annotations>
+            <features value="AdobeStockImagePanel"/>
+            <stories value="Access Adobe Stock Image panel via the Admin"/>
+            <title value="User searches stock images by keywords"/>
+            <description value="User searches stock images by keywords"/>
+            <severity value="CRITICAL"/>
+            <group value="AdobeStockIntegration"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminAdobeStockGoToConfigurationPageActionGroup" stepKey="goToAdobeStockConfigurationPage"/>
```

Comment:
Please, check my comment here https://github.com/magento/adobe-stock-integration/pull/239/files#r305875535
If it's possible, we should set the configuration values in a preferred way in order to avoid additional overhead.

Reply:
@rogyar If you record via the config:set - adobe api key and private key will be hardcoded, it is not suitable for our case.

Reply:
Makes sense. However, we can have as many key pairs using different entities as we need. So, in this case no matter where exactly the values will be hardcoded in a test body or in the entities set. 

If we still need to put dynamic values in every test we can try passing `apiKey` and `privateKey` to a separate action group and calling `config:set` from the action group.

Reply:
Makes sense. However, we can have as many key pairs using different entities as we need. So, in this case no matter where exactly the values will be hardcoded in a test body or in the entities set.

If we still need to put dynamic values in every test we can try passing apiKey and privateKey to a separate action group and calling config:set from the action group.